### PR TITLE
Feature/food items/upload photo

### LINKED ISSUE #21: As a user, I want to upload an image to an item
"So much text I can enter, but no image???"

- backend
  - handle MultiPartFile and upload to Cloudinary
- frontend
  - add picture selector to form and handle data
- unit and integration tests

### PR DESCRIPTION
**Pull request by valfink**
Resolve #21

### CODE REVIEW

File: frontend/src/component/FoodItemForm.tsx
Comment (u-dwin) [inline frontend/src/component/FoodItemForm.tsx:1]: I feel you could outsource a lot of the functions into a hook for a better overview. But it's also alright as is.

**Review by u-dwin — APPROVED** [review]
Nice! I do not see any issues with the code. I tried to run it though and whenever I want to add a food item I receive a bad request error 500. Please ensure that it is working on your side. It might be that there is something missing on my side (MongoDB is connected). Feel free to reach out if you have any questions.

**Comment by valfink** [comment]
Thanks for the feedback! Which browser are you using? Did you manage to register and log in successfully?

**Comment by u-dwin** [comment]
Sorry, only saw your comment now, I used Chrome. Registration and Login worked for me.

**Pull request merged**